lib/elixir/lib/kernel/special_forms.ex

```diff
@@ -808,15 +808,17 @@ defmodule Kernel.SpecialForms do
   quoted. They are:
 
       :sum         #=> Atoms
-      1            #=> Integers
-      2.0          #=> Floats
+      1            #=> Unsigned integers (*)
```

Comment:
What if we include the explanation here?
```suggestion
      1            #=> Integers without the + or - sign
```